stable/prometheus-node-exporter/templates/serviceaccount.yaml

```diff
@@ -1,15 +1,13 @@
-{{- if .Values.rbac.create -}}
```

Comment:
please set this value as true on values.yaml

Reply:
Hey @gianrubio, I already see rbac.create as true in values.yaml
https://github.com/helm/charts/blob/master/stable/prometheus-node-exporter/values.yaml#L50

Reply:
@gianrubio ping :)